cmd/manager/main.go

```diff
@@ -314,6 +314,8 @@ func serveCRMetrics(cfg *rest.Config) error {
 func reconcilePrometheusRule(promethuesRule *monitoringv1.PrometheusRule) {
 	labels := map[string]string{
 		"monitoring-key": "middleware",
+		"prometheus":     "application-monitoring",
```

Comment:
I'm adding these because they existed in previous version. I don't think they are necessary.

Reply:
I remember discussing this with @pb82 a while back as well. I think it was a different operator. He may be able to confirm if removing these will break anything.